lib/app_component.dart

```diff
@@ -49,7 +50,6 @@ class AppComponent {
 //  ];
 
   final RegExp nonNumberMatcher = new RegExp(r"[^0-9]");
```

Comment:
Will look into it. Due to the changes in this PR this matcher will probably be factored out actually.